app/assets/javascripts/discourse/app/lib/sidebar/user/community-section/my-posts-section-link.js

```diff
@@ -98,11 +98,11 @@ export default class MyPostsSectionLink extends BaseSectionLink {
     return this.draftCount > 0;
   }
 
-  get defaultPrefixValue() {
+  get prefixValue() {
```

Comment:
@lis2 @tgxworld without this change, the override doesn't work, it defaults to "user" regardless. I think that's because we pass an icon value, so the `defaultPrefixValue` isn't evaluated. 

The documentation isn't very clear either: https://github.com/discourse/discourse/blob/main/app/assets/javascripts/discourse/app/lib/sidebar/base-community-section-link.js#L108-L122

Reply:
This is tricky because all the community links are considered to have its icon overridden

https://github.com/discourse/discourse/blob/2d867aa8e5f3db8bfa251fb384b96dfb477b3d8e/app/models/sidebar_url.rb#L9-L44

This is why `defaultPrefixValue` is no longer working.